examples/rcbd_two_var.R

```diff
@@ -0,0 +1,118 @@
+
+# Construct the model.
+#-----------------------------------------------------------------------------#
+model <- aov(yield ~ block + clone + nitrogen + clone:nitrogen, data=my.data)
+#-----------------------------------------------------------------------------#
+
+# Plot the four standard fit plots: residuals vs predicted, sqrt of residuals vs
+# fitted, Normal Q-Q plot of the residuals, residuals vs leverages.
+dev.new()
+#------------------------------------------------------------------------------#
+par(mfrow = c(2, 2), oma = c(0, 0, 2, 0))
+plot(model)
+#------------------------------------------------------------------------------#
+
+# Make sure the residuals are normal (this can also be seen in the Q-Q plot).
+cat('Shapiro-Wilk Normality Test\n')
+sep(50)
+#------------------------------------------------------------------------------#
+shapiro.test(residuals(model))
+#------------------------------------------------------------------------------#
+sep(50)
+
+# Run Levene's Test.
+# TODO : This gives the error: "Model must be completely crossed formula only."
+# Levene's test only valid for 1-way ANOVA. thus, we run levene's for each factor we are interested in.
+# in this case: clone, nitrogen 
+cat("Levene's Test\n")
+sep(79)
+#------------------------------------------------------------------------------#
+#leveneTest(model)
+leveneTest(yield ~ clone, data=my.data)
+leveneTest(yield ~ nitrogen, data=my.data)
+#Note: clone is significant in Levene's test so transformation would be necessary
+#------------------------------------------------------------------------------#
+sep(79)
+
+# The user will select the alpha level from the app.
+#------------------------------------------------------------------------------#
+alpha <- 0.05
+#------------------------------------------------------------------------------#
+
+# Print the ANOVA table of the fit. The user should will have to note the
+# significant factors. In this case all factors, including interaction nitrogen*clone, are significant.
+cat('ANOVA Table\n')
+sep(50)
+#------------------------------------------------------------------------------#
+anova(model)
+#------------------------------------------------------------------------------#
+sep(50)
+
+# Show the confidence intervals for the intercept and each virus level.
+cat('Confidence Intervals\n')
+sep(50)
+#------------------------------------------------------------------------------#
+confint(model, level=1.0 - alpha)
+#------------------------------------------------------------------------------#
+sep(50)
+
+# Plot the mean yield with respect to each virus level.
+# TODO : Is the default plot ok here, or is something different desired?
+dev.new()
+#------------------------------------------------------------------------------#
+plot(allEffects(model))
+#------------------------------------------------------------------------------#
+
+# The ANOVA table shows that each variable and the interaction are significant,
```

Comment:
Note, if this were not the case (i.e., not a significant interaction) the "main effects" could be tested with the LSD:
LSD.test(model, ....will fill in later....